pkg/util/arangod/client.go

```diff
@@ -200,6 +218,22 @@ func createArangodHTTPConfigForDNSNames(ctx context.Context, cli corev1.CoreV1In
 	return connConfig, nil
 }
 
+// createArangodVSTConfigForDNSNames creates a go-driver VST connection config for a given DNS names.
+func createArangodVSTConfigForDNSNames(ctx context.Context, cli corev1.CoreV1Interface, apiObject *api.ArangoDeployment, dnsNames []string) (http.ConnectionConfig, error) {
+	scheme := "vst"
```

Comment:
Should be "http". There is no "vst" scheme in the go-driver, just a VST connection that re-uses http/https schemes.